docs/end-to-end-demo.md

```diff
@@ -77,8 +77,11 @@ carry out one or the other, then continue on with the rest of the steps.
     ```
 
 ## Set up mock udev video devices
-1. Install a kernel module to make v4l2 loopback video devices. Learn more about this module [here](https://github.com/umlaeute/v4l2loopback).
+1. On a fresh Ubuntu (20.04 LTS for example), first update packages already installed. Then, install a kernel module to make v4l2 loopback video devices. It comes with prequisites: the other kernel module 'videodev' part of package linux-modules-extra and also the dkms package (Dynamic Kernel Module Support). Learn more about v4l2 loopback [here](https://github.com/umlaeute/v4l2loopback)
```

Comment:
It's not required that this demo is run on Ubuntu. Only if they are using MicroK8s later. I think we can add some background about obtaining an environment before this section in a later PR. Can we simplify this to "To make dummy video4linux devices, install the v4l2loopback kernel module and its prerequisites. Learn more about ... "

Reply:
@kate-goldenring : Kate. see last commit adapting text as per your suggestion